examples/python/ocr/README.md

```diff
@@ -0,0 +1,156 @@
+
+<picture>
+  <img src="https://static.rerun.io/ocr1/54b3a9d0706fd4a3a3dcbf878046ae34a7a6feec/full.png" alt="">
+  <source media="(max-width: 480px)" srcset="https://static.rerun.io/ocr1/54b3a9d0706fd4a3a3dcbf878046ae34a7a6feec/480w.png">
+  <source media="(max-width: 768px)" srcset="https://static.rerun.io/ocr1/54b3a9d0706fd4a3a3dcbf878046ae34a7a6feec/768w.png">
+  <source media="(max-width: 1024px)" srcset="https://static.rerun.io/ocr1/54b3a9d0706fd4a3a3dcbf878046ae34a7a6feec/1024w.png">
+  <source media="(max-width: 1200px)" srcset="https://static.rerun.io/ocr1/54b3a9d0706fd4a3a3dcbf878046ae34a7a6feec/1200w.png">
+</picture>
+
+
+## Used Rerun types
+[`Image`](https://www.rerun.io/docs/reference/types/archetypes/image), [`TextDocument`](https://rerun.io/docs/reference/types/archetypes/text_document), [`Boxes2D`](https://rerun.io/docs/reference/types/archetypes/boxes2d), [`AnnotationContext`](https://rerun.io/docs/reference/types/archetypes/annotation_context)
+
+## Background
+This example demonstrates the ability to visualize and verify the document layout analysis and text detection using the [PaddleOCR](https://github.com/PaddlePaddle/PaddleOCR).
+[PP-Structure](https://github.com/PaddlePaddle/PaddleOCR/tree/main/ppstructure) used for this task, which is an intelligent document analysis system developed by the PaddleOCR team, which aims to help developers better complete tasks related to document understanding such as layout analysis and table recognition.
+In the layout analysis task, the image first goes through the layout analysis model to divide the image into different areas such as text, table, figure and more, and then analyze these areas separately.
+The classification of layouts and the text detection (including confidence levels) are visualized in the Rerun viewer.
+Finally, the recovery text document section presents the restored document with sorted order. By clicking on the restored text, the text area will be highlighted.
+
+## Logging and visualizing with Rerun
+The visualizations in this example were created with the following Rerun code.
+
+### Image
+The input document is logged as [`Image`](https://www.rerun.io/docs/reference/types/archetypes/image) object to the `Image` entity.
+```python
+rr.log("Image", rr.Image(coloured_image))
+```
+
+### Label mapping
+
+An annotation context is logged with a class ID and a color assigned per layout type using [`AnnotationContext`](https://rerun.io/docs/reference/types/archetypes/annotation_context).
+
+```python
+class LayoutType(Enum):
+    UNKNOWN = (0, "unknown", Color.Purple)
+    TITLE = (1, "title", Color.Red)
+    TEXT = (2, "text", Color.Green)
+    FIGURE = (3, "figure", Color.Blue)
+    FIGURE_CAPTION = (4, "figure_caption", Color.Yellow)
+    TABLE = (5, "table", Color.Cyan)
+    TABLE_CAPTION = (6, "table_caption", Color.Magenta)
+    REFERENCE = (7, "reference", Color.Purple)
+    FOOTER = (7, "footer", Color.Orange)
```

Comment:
is it intentional that they both have the same number?
(same in the actual code)

Reply:
Noo, thanks for noticing. Can you change that? Footer could be 8

Reply:
k!